hash.ts

```diff
@@ -0,0 +1,12 @@
+import { crypto } from "https://deno.land/std/crypto/mod.ts";
```

Comment:
I suppose it's probably about time for a `deps.ts` 😄  Think you could move these there?

Reply:
yup, I considered it.

Maybe an action with a fmt check also? So lazy people like me don't need you to remind me to run fmt.